src/gui/MainWindow.cpp

```diff
@@ -822,18 +826,24 @@ void MainWindow::onActiveDocumentChanged(const int newIndex){
             for(QAction * action:singleViewAct) action->setChecked(false);
             singleViewAct[documents[activeDocumentId]->getDisplayGrid()->getActiveDisplayId()]->setChecked(true);
         }
-
+    }
+    else if (displayGrid == nullptr && documentArea->count() > 1) {
+        objectTreeWidgetDockable->clear();
+        objectPropertiesDockable->clear();
+        statusBarPathLabel->setText("");
+        activeDocumentId = -1;
     }
 }
 
 void MainWindow::tabCloseRequested(const int i)
 {
     documentArea->removeTab(i);
-    if (documentArea->currentIndex() == -1){
+    if (documentArea->currentIndex() == -1 || QString::compare(documentArea->tabText(documentArea->currentIndex()), "Quick Start", Qt::CaseSensitive) == 0) {
         objectTreeWidgetDockable->clear();
         objectPropertiesDockable->clear();
+        statusBarPathLabel->setText("No document open");
+
```

Comment:
I would keep the status bar empty here. You want to use it for other things anyway.